docs/devguide/fields-yml.asciidoc

```diff
@@ -29,27 +30,36 @@ Define the field mappings in the `fields` array:
 ----------------------------------------------------------------------
 
 <1> `name`: The field name
-<2> `type`: The field type. The value for the `type` key can be any of the datatypes https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-types.html[available in Elasticsearch]. If no value is specified, the field will default to being a `keyword`.
+<2> `type`: The field type. The value of `type` can be any datatype {ref}/mapping-types.html[available in {es}]. If no value is specified, the default type is `keyword`.
 <3> `required`: Whether or not a field value is required
 <4> `description`: Some information about the field contents
 
 ==== Mapping parameters
-Other mapping parameters can be specified for each field. Consult the https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-params.html[Elasticsearch reference] for more details on each of these parameters.
+
+You can specify other mapping parameters for each field. See the
+{ref}/mapping-params.html[{es} Reference] for more details about each
+parameter.
 
 [horizontal]
 `format`:: Specify a custom date format used by the field.
-`multi_fields`:: For `text` or `keyword` fields, `multi_fields` can be used to define multi-field mappings.
+`multi_fields`:: For `text` or `keyword` fields, use `multi_fields` to define
+multi-field mappings.
 `enabled`:: Whether or not the field is enabled.
 `analyzer`:: Which analyzer to use when indexing.
 `search_analyzer`:: Which analyzer to use when searching.
 `norms`:: Applies to `text` and `keyword` fields. Default is `false`.
-`dynamic`:: Dynamic field control. Can be one of `true` (default), `false` or `strict`.
+`dynamic`:: Dynamic field control. Can be one of `true` (default), `false`, or
+`strict`.
 `index`:: Whether or not the field should be indexed.
-`doc_values`:: Whether or not the field should have doc values generated. See docs.
+`doc_values`:: Whether or not the field should have doc values generated.
 `copy_to`:: Which field to copy the field value into.
-`ignore_above`:: When this property value is missing or is `0`, it receives the `libbeat` default value of `1024`. If the value is `-1`, the Elasticsearch default value will be applied. Any other specified value will be applied as-is.
+`ignore_above`:: {es} ignores (does not index) strings that are longer than the
```

Comment:
@ycombinator Does this look OK? I want to map the definition to the setting name, but also explain more clearly what it does. BTWe, I also added "characters" in line 58 because I don't think it's clear from the description that this is a character count.

Reply:
Yes, this looks good, thanks!